src/window.rs

```diff
@@ -1396,6 +1396,19 @@ impl Window {
             .maybe_wait_on_main(|w| w.drag_resize_window(direction))
     }
 
+    /// Show window menu at current mouse position.
+    ///
```

Comment:
```suggestion
    ///
    /// This is the context menu that is normally shown when interacting with
    /// the title bar. This is useful when implementing custom decorations.
```
@kchibisov is this enough? Or do you want do add more information here?